CHANGELOG.md

```diff
@@ -42,6 +42,13 @@
     - `-experimental.tsdb.flush-blocks-on-shutdown` changed to `-experimental.blocks-storage.tsdb.flush-blocks-on-shutdown`
 * [CHANGE] Flags `-bigtable.grpc-use-gzip-compression`, `-ingester.client.grpc-use-gzip-compression`, `-querier.frontend-client.grpc-use-gzip-compression` are now deprecated. #2940
 * [CHANGE] Limit errors reported by ingester during query-time now return HTTP status code 422. #2941
+* [CHANGE] Cache: added support for Redis Cluster and Redis Sentinel. #2961
```

Comment:
We're cutting Cortex `1.3.0` and this PR will be merged after that. Please rebase `master` and move the CHANGELOG entry under the `master / unreleased` section at the top of the CHANGELOG. Thanks!